### ISSUE TITLE
using ST7789 library for Mini Display Hat... How do I control the onboard LED?

### ISSUE THREAD

**Issue by Nwhitten** [pimoroni/st7789-python#20]
Hi

when using the ST7789 library (not the displayhatmini library) is there a way to control the onboard LED??

thanks

Neil

**Comment by Sebwap** [comment]
Yes probably, by replicating the code included in display hat mini library!

**Comment by Wikinaut** [comment]
see https://github.com/Wikinaut/pinetradio/blob/main/ledtest.py

**Comment by Wikinaut** [comment]
You do not need the library. Just control the `ACT` LED on pin 47.

See for example lines in https://github.com/Wikinaut/pinetradio/blob/main/pinetradio.py (look for `ACT`, which stands for activity LED.).

PI system use of the ACT-LED must be disabled. You can use this service https://github.com/Wikinaut/pinetradio/blob/main/disable-led.service and run `sudo systemctl enable disable-led` once.
